### DISCUSSION TITLE
关于30分钟笔中枢一买的讨论

### DISCUSSION THREAD

**Discussion by chtone** [waditu/czsc#26] [Ideas]
30分钟第一买点的要求：
1、连续两个30分钟笔中枢下跌。
2、跌破最后一个中枢最低点后，
      后一根K线成交量比前一根K线成交量放大1.5倍以上。
3、同时这根放巨量的K线或者后一、二根K线收出较长下影线。
4、买点：在收出下影线这根K线收盘前三分钟买入，或者在下一根K线开盘买入。

其他技术指标配合：
1、RSI指标：在收出下影线这根K线出现时，RSI白线值<=18
2、MACD指标：对比中枢前下跌，本次下跌力度明显减小。
3、不买ST类股。
4、任何一次交易开始之前，都要有明确的操作计划表，不能随心买卖。

止盈、止损点：
1、止损点：长下影线那根K线的最低点低两个价位止损。
2、止盈位1：最后一个中枢底边放量（后一根K线是前一根的成交量的1.5以上）止盈50%
       止盈位2：最后一个中枢顶边放量（后一根K线是前一根的成交量的1.5以上）止盈50%

大家看看这个简单的策略有没有可行性，谢谢！图片不知道怎么上传，大家想象一下吧，两个笔中枢下跌。^_^

**Comment by zengbin93** [comment]
我来补一下图片：

![image](https://user-images.githubusercontent.com/24988472/101848350-34acc780-3b90-11eb-85d6-74b49290114a.png)

![image](https://user-images.githubusercontent.com/24988472/101848381-4b531e80-3b90-11eb-8c5f-549ca1ab0808.png)

![image](https://user-images.githubusercontent.com/24988472/101848432-70e02800-3b90-11eb-9e5d-bb4918d4b066.png)

**Comment by tconsky** [comment]
1. 双中枢要出了背驰结构
2. 出现底分型
3. 剔除掉近期有解禁或者股东减持的票

**Comment by skanda-huayan** [comment]
如果用a + A + b + B + c，来表示这个下跌：
1 或者严格满足趋势背驰的定义，在c内部有最后一个中枢B的三卖。如果c的内部结构还背驰，极佳。
这个一买可以做一个走势；
2 或者只是c与b盘整背驰，相当于趋势中的盘整背驰一买，c的内部没有B的三卖，只能做向上一段。
这两种情况的多，级别是不一样的。所以止盈的策略不一样。
不管怎样，至少都要操作级别和次级别同时结合起来判断。
